src/ape_plugins/_cli.py

```diff
@@ -168,7 +168,38 @@ def install(cli_ctx, skip_confirmation):
             # NOTE: Be *extremely careful* with this command, as it modifies the user's
             #       installed packages, to potentially catastrophic results
             # NOTE: This is not abstracted into another function *on purpose*
-            subprocess.call([sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"])
+            result = subprocess.call(
+                [sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"]
+            )
+            if result != 0:
```

Comment:
We don't want to exit per plugin, we want to exit at the very end (outside the for loop) with status of 1 if any of them have failed